Changed uin8 to uint8 in response to issue #9266

CoolRanch29 opened the pull request:
closes #9266

sinhrks committed:
ENH: drop function now has errors keyword for non-existing column handling

sinhrks committed:
BUG: GroupBy.get_group raises ValueError when group key contains NaT

jorisvandenbossche committed:
DOC: remove mention of TimeSeries in docs

sinhrks committed:
ENH: Add BusinessHour offset

Commit:
Fix to allow sparse dataframes to have nan column labels

Support for nan columns

Fix

Trigger Travis CI

jreback fixes

Release note update

Commit:
ENH: Allow get_dummies to return sparse dataframe

ENH: Allow get_dummies to return sparse dataframe

ENH: Allow get_dummies to return sparse dataframe

Fix

Fix

Fixes

Bug in order of columns

Slight speed improvement

get_dummies update

Release notes update

Remove convert dummies test

tiagoantao committed:
Applying max_colwidth to the DataFrame index (#7856)

justification to comply with tests

jlec committed:
ENH: Use datetutil.tz.gettz() instead of dateutil.zoneinfo.gettz()

python-dateutil provides two implementations for gettz(), tz.gettz() and
zoneinfo.gettz(). The former tries first to use system provided timezone data,
where as the later always uses a bundled tarball. Upstreams recommandation
for library consumers is only using tz.gettz() (1 & 2). Further more, on
system which do not install the zoninfo tarball (e.g. Debian, Gentoo and
Fedora) but rely on the system zoneinfo files the direct usage of
zoneinfo.gettz() creates problems which result in test failures (3 - 6).

For compatibility in pandas code

    pandas.tslib._dateutil_gettz()

should be used.

1 https://github.com/dateutil/dateutil/issues/8
2 https://github.com/dateutil/dateutil/issues/11
3 https://github.com/pydata/pandas/issues/9059
4 https://github.com/pydata/pandas/issues/8639
5 https://github.com/pydata/pandas/issues/10121
6 https://github.com/pydata/pandas/issues/9663

Signed-off-by: Justin Lecher <jlec@gentoo.org>

CoolRanch29 committed:
Changed uin8 to uint8 in response to issue #9266

shoyer commented:
Thanks, this needs:

1. a test case which breaks before this change (you can use the example from #9266; add it to `pandas/io/tests/test_parsers.py`)
2. an addition the "bug fixes" section of `docs/sources/whatsnew/0.16.0.txt` (refer to the original issue)

CoolRanch29 committed:
Added test for GH 9266

sinhrks committed:
DOC: Revise merging.rst with graphical examples

CoolRanch29 commented:
This is my first PR attempt and I don't understand this error message.  Did I do something wrong in the test function I added?

shoyer commented on pandas/io/tests/test_parsers.py:
```diff
@@ -2984,6 +2984,28 @@ def test_variable_width_unicode(self):
+        with open('test.txt','w') as f:
+            f.write(
+            """1421302964.213420    PRI=3 PGN=0xef00      DST=0x17 SRC=0x28    04 154 00 00 00 00 00 127
+               1421302964.226776    PRI=6 PGN=0xf002               SRC=0x47    243 00 00 255 247 00 00 71"""
```
I think the issue is that this line starts with extra whitespace, throwing off `colspecs` below.

CoolRanch29 committed:
Trying again with new testing function

Commit:
ENH: Don't infer WOM-5MON if we don't support it (#9425)

sinhrks committed:
DOC/CLN: Revise StringMethods docs

Commit:
Moved caching in `AbstractHolidayCalendar` to the instance level
Closes #9552

jreback commented:
@CoolRanch29 I rebase and push your branch up here. pls review and update this PR when you can
https://github.com/jreback/pandas/tree/CoolRanch29-master

I am not sure what you are trying to compare against here (e.g. what should the expected frame look like?)

mortada committed:
improve error message when importing pandas from source directory

nickeubank committed:
ENH: Add sample function with tests and docs (GH2419)

Commit:
ENH: Allow clip, clip_lower, and clip_upper to use array-like thresholds (GH 6966)

mortada committed:
ENH: add StringMethods (.str accessor) to Index, fixes #9068

Commit:
Fixed bug #9671 where 'DataFrame.plot()' raised an error when both 'color' and 'style' keywords were passed and there was no color symbol in the style strings (this should be allowed)

CoolRanch29 commented:
Okay, I added an expected frame to test again.  The problem before is that pd.read_fwf wasn't working at all - an exception was thrown when trying to read the test case.  I'm still concerned that in the resulting frame, the column 'dst' is of type 'object' when using read_fwf and of type 'float64' when created by DataFrame().

jreback commented on doc/source/whatsnew/v0.16.0.txt:
```diff
@@ -209,4 +209,9 @@ Bug Fixes
 - Fixes issue with ``index_col=False`` when ``usecols`` is also specified in ``read_csv``. (:issue:`9082`)
 - Bug where ``wide_to_long`` would modify the input stubnames list (:issue:`9204`)
+<<<<<<< HEAD
+
```
needs fixing here (eliminate the merge lines)

jreback commented on pandas/io/tests/test_parsers.py:
```diff
@@ -2984,6 +2984,31 @@ def test_variable_width_unicode(self):
+        'dst':lambda x: int(x,16),
+        'data':lambda x: len(x.split(' '))},
+        index_col='time')
+        expected = pd.DataFrame([[ 3, 61184, 23, 40, 8] ,
```
this doesn't look property formatted to me

JonasAbernot committed:
fix issue #9680

Commit:
Fixes groupby.apply() error when no returns #9684

jreback commented:
whats new is now available for 0.16.1 https://github.com/pydata/pandas/commit/5ebf521eac528f5a4b772e4677f70b689b8b1826

danbirken committed:
BUG: Fix error when reading postgres table with timezone #7139

`read_sql_table()` will break if it reads a table with a `timestamp
with time zone` column if individual rows within that column have
different time zones. This is very common due to daylight savings time.

Pandas right now does not have good support for a Series containing
datetimes with different time zones (hence this bug).  So this change
simply converts a `timestamp with time zone` column into UTC during
import, which pandas has great support for.

jorisvandenbossche committed:
DOC: fix api documentation for accessors

evanpw committed:
ENH: Add an axis parameter to DataFrame.diff

jreback committed:
support CategoricalIndex

raise KeyError when accessing invalid elements
setting elements not in the categories is equiv of .append() (which coerces to an Index)

davidastephens committed:
BUG: raw_locales unreachable in util.testing.get_locales

dsm054 committed:
BUG: DataFrame.equals should not care about block order (GH #9330)

evanpw committed:
ENH: Allow conversion of datetime64 and timedelta64 to string in astype (GH 9757)

evanpw committed:
ENH: Add option in read_csv to infer compression type from filename

jorisvandenbossche committed:
DOC: fix some various doc warnings

sinhrks committed:
ENH: Add StringMethods.partition and rpartition

invisibleroads committed:
Fix zlib and blosc imports

invisibleroads committed:
Add missing keys and tests

invisibleroads committed:
Make test_plain actually plain

invisibleroads committed:
Add blosc to .travis.yml

invisibleroads committed:
Add blosc to requirements

invisibleroads committed:
Diagnose why blosc is not importing

invisibleroads committed:
Get travis to cooperate

invisibleroads committed:
Make travis listen

invisibleroads committed:
Respond to comments from jreback

invisibleroads committed:
Restore spacing

sinhrks committed:
API: define _constructor_expanddim for subclassing Series and DataFrame

kshedden committed:
Closes #9795 (Stata writer changes input frame)

Add note to release notes

evanpw committed:
BUG: where behaves badly when dtype of self is datetime or timedelta, and dtype of other is not (GH9804)

evanpw committed:
DOC: Fix broken formatting on docstring examples with first-line comments

tomazberisa committed:
BUG: Fix for #9764

Values from range [1e-7, 5e-7] (for display.precision=7) not displaying 0 anymore

tomazberisa committed:
TST: Test for #9764 fix

tomazberisa committed:
TST: Test saves and restore context (#9764)

jreback committed:
DOC: add dev environment creation details to contributing.rst

jreback commented:
can you rebase this?

tomazberisa committed:
CLN: Test code cleanup (#9764)

tomazberisa committed:
DOC: Update whatsnew for 0.16.1 (#9764)

CoolRanch29 commented:
I'm pretty sure I just rebased.  I'm sorry if this is causing a headache for you, I'm sure I'm doing some things stupidly.  I've gotten a lot of benefit out of using pandas and so I'm trying to contribute but this is my first time contributing so I'm having to learn a lot about both the pandas code and github.

jreback commented on pandas/io/tests/test_parsers.py:
```diff
@@ -2984,6 +2984,28 @@ def test_variable_width_unicode(self):
 
+    def test_convert_to_nd_arrays(self):
+        #GH 9266
+        with open('test.txt','w') as f:
```
use
```
with tm.ensure_clean('test.txt') as path:
     with open(path,'w') as f:
          ....
```

jreback commented on pandas/io/tests/test_parsers.py:
```diff
@@ -2984,6 +2984,28 @@ def test_variable_width_unicode(self):
+                """1421302964.213420    PRI=3 PGN=0xef00      DST=0x17 SRC=0x28    04 154 00 00 00 00 00 127 \n"""
+            +   """1421302964.226776    PRI=6 PGN=0xf002               SRC=0x47    243 00 00 255 247 00 00 71"""
+                )
+        try:
```
do 

```
result = pd.read_fwf(......)
self.assertEqual(result['dst'].dtype,np.uint8)
```

(I for whichever columns should be that dtype)

jreback commented:
couple of style things. pls add a release note in v0.16.1 as well.

TomAugspurger committed:
API: Sort keys for DataFrame.assign

Previously the order was arbitrary. For predicitability,
we'll sort before inserting.

CoolRanch29 committed:
Modified the test_to_nd_array test to make it clearer what was being tested.

CoolRanch29 committed:
Added documentation for bug fix on issue #9266

jreback committed:
DOC: correction to contributing.rst

ssanderson committed:
BUG: Preserve tz of start_date in Holiday.dates

Previously, the timezone of only the start_date was lost, causing the
subsequent call to `DatetimeIndex` to fail if a both start_date and
end_date were tz-aware.

evanpw committed:
DOC: Clean up documentation for convert_objects

Commit:
TST: Fix tests in TestGoogle

evanpw committed:
BUG: skiprows doesn't handle blank lines properly when engine='c' (GH #9832)

Commit:
BUG: Issue 9798 fixed

BUG: #9798 `index_col` shouldn't accept the value `True`

move the check to ``TextFileReader`` ``clean_options``

evanpw committed:
BUG: read_csv skips lines with initial whitespace + one non-space character (GH9710)

sinhrks committed:
DOC: Fix release note for v0.16

behzadnouri committed:
memory access bug in read_csv causing segfault

evanpw committed:
Add tests with time zones, and clean up DataFrame test

jankatins committed:
Fix: unequal comparisons of categorical and scalar

Before, unequal comparisons were not checking the order of the
categories.

This was due to a conversion to an ndarray, which turned the
comparison to one between ndarray and scalar, which of course
has no categories to take into account.

Also add test cases and remove the one which actually tested the
wrong behaviour.

sinhrks committed:
BUG: plot(kind=hist) results in TypeError for non-numeric data

sinhrks committed:
BUG/CLN: Repeated time-series plot may raise TypeError

sinhrks committed:
BUG: secondary ax has incorrect right_ax property

mortada committed:
DOC: add more examples to StringMethods on Index

jreback committed:
DOC: add note about hosted dev docs

mortada committed:
DOC/CLN: fixed several typos in categorical.rst

jankatins committed:
BUG: Fix for comparisons of categorical and an scalar not in categories, xref GH9836

Up to now, a comparison of categorical data and a scalar, which
is not in the categories would return `False` for all elements when
it should raise a `TypeError`, which it now does.

Also fix that `!=` comparisons would return `False` for all elements
when the more logical choice would be `True`.

mortada committed:
DOC/CLN: fixed bloolean indexing example, cleaned up typos

jreback committed:
wip

chrisgilmerproj committed:
GH9570 allow timedelta string conversion without leading zero

ajamian committed:
BUG: Fix for cython version checking #9827; adding appropriate travis
builds to test.

jcrist committed:
Period accepts datetime64 value

Added support for `datetime64` value for Period. Fixes #9054.

hsperr committed:
FIX: division of Decimal would crash on fill because Decimal does not support type or dtype. (GH9787)
ENH: replace np.isscalar with better lib.isscalar
ADD: Test decimal division

pykler committed:
BUG: Fixing == __eq__ operator for MultiIndex ... closes (GH9785)

jcrist committed:
Groupby transform preserves output dtype

Previously `transform` output was always the same dtype as the groupby
object. This allows the output dtype to differ from the input. Fixes #9807.

scari committed:
BUG: unstack on unicode name level breaks GH9856

[PYCON 2015 Sprints]

ksanghai committed:
added the fix for index name lost #9862 and also added the corresponding test in
test_index

cpcloud committed:
Whitespace

cpcloud committed:
Fix rhs dict assignment

hsperr committed:
FIX: timeseries asfreq would drop the name of the index, closes #9854

cpcloud committed:
What's new and doc

hoh committed:
FIX: Describe NotImplementedErrors or use AbstractMethodError

This issue fixes #7872 by either describing the reason why a NotImplementedError is raised,
or by throwing an AbstractMethodError in the case of abstract methods (eg: Mixins) that
should be overwritten by the inheriting classes.

This issue is based on discussion of PR #7899 which was never merged.

cpcloud committed:
Fix blosc compressor test

hsperr committed:
ENH: Raise error on trying to write excel file with a MultiIndexed DataFrame. closes #9794

mortada committed:
ENH: allow Panel.shift on items axis

mortada committed:
add/delete str/dt/cat dynamically from __dir__ (fix for #9627)

cpcloud committed:
Fix 32 syntax u error

cpcloud committed:
Add tests for more units

jreback committed:
move blosc import to install_conda.sh script

hsperr committed:
ENH: repr now shows index name #6482

move tests to generically tests for index
generify __unicode__ for Index

adjust index display to max_seq_items

Commit:
Fix of the docs for tz_conver and tz_localize

yred committed:
BUG: Fixed latex output for multi-indexed dataframes - GH9778

jorisvandenbossche committed:
DOC: add redirects to ensure old links instead of building the docstring page

jreback committed:
formatting MultiIndex

sinhrks committed:
BUG: barplot with log=True not working for values smaller than 1

sinhrks committed:
TST: Check index names

nburns committed:
removed conditional

ajamian committed:
TST: adding sdist and install tests to the 2.6 regression - closes #9878

sinhrks committed:
BUG: Index.name is lost during timedelta ops

sinhrks committed:
BUG: GroupBy.size doesnt attach index name properly if grouped by TimeGrouper

Commit:
Update README.md

Travis build status badge should be clickable.

TomAugspurger committed:
BUG: hidden ticklabels with sharex and secondary

evanpw committed:
BUG: Exception when setting an empty range using DataFrame.loc

toobaz committed:
DOC: Link to recipe for SafeHDF5Store

jreback committed:
PERF: improve perf of writing csv's with datetimes

evanpw committed:
BUG: DataFrame constructor fails when columns is set and data=[] (GH9948/9939)

jreback committed:
perf improvements for other native type writers

jreback committed:
BUILD: add bz2, gz to setup.py, xref GH9770

jreback committed:
TST: skip testing locales on win32, xref GH9845

jreback committed:
TST: dtype on comparison for test_panel_to_expanddim

jreback committed:
dtype fix for GH9804 on win32

jreback committed:
DOC: fix incorrect issue numbers in whatsnew

jreback committed:
fixup for CategoricalIndex merge

increase limits for max_seq_items & printing for Index

add extended repr for datetimelike indexes

fix tseries/test_base for repr

adjust docs for repr-name

use new format_data on all Index types

jreback committed:
DOC: fixup whatsnew for rebase issue

ptype committed:
Spelling typo

documenation -> documentation

mortada committed:
CLN: remove an unnecessary warning from test_index.py

mortada committed:
Allow add_categories() to accept Series/np.array

sinhrks committed:
DOC: Release note link and format correction

mortada committed:
DOC: StringMethods html doc fixes

Commit:
BUG: Make .iloc and .loc indexing consistent on empty dataframes

Tests

Fix

Test reorder

Doc update

Tests fix

Tests fix

SQL tests fix

Testing update

Fixes

Testing fix

Test fix

jorisvandenbossche committed:
DOC: provide more links to API documentation where possible (GH3705)

Part 1: 10min and basics.rst

jorisvandenbossche committed:
DOC: clean up / consistent imports (GH9886)

mortada committed:
ENH: improve extract and get_dummies methods for Index.str (fix for #9980)

simplify str_extract(), pass name into _wrap_result()

mortada committed:
DOC/CLN: HTML docs fixes

evanpw committed:
BUG: transform and filter misbehave when grouping on categorical data (GH 9921)

slonik-az committed:
BUGFIX: length_of_indexer() can return incorrect values that break slice assignments.

BUG DESCRIPTION:
length_of_indexer() (defined in pandas/core/indexing.py) returns incorrect result
if (stop-start) is not divisible by step. As a consequence some slice assignments
throw exceptions. Affected panda versions: 0.16.x, 0.15.x, current git master.

HOW TO REPRODUCE:

import pandas as pd
sr= pd.Series([-1]*6) # series with 6 elements indexed from 0 to 5.
sr[0::2]= [0,2,4] # setting even elements works fine!
sr[1::2]= [1,3,5] # setting odd elements results in error:

.../pandas/core/internals.pyc in setitem(self, indexer, value)
    568             if is_list_like(value) and l:
    569                 if len(value) != length_of_indexer(indexer, values):
--> 570                     raise ValueError("cannot set using a slice indexer with a "
    571                                      "different length than the value")
    572
ValueError: cannot set using a slice indexer with a different length than the value

slonik-az committed:
Add bugfix note to whatsnew in v0.16.1 release.

jreback commented:
We'd like to merge this. Pls address the comments from above. (and needs a rebase). see [here](http://pandas-docs.github.io/pandas-docs-travis/contributing.html) for help on this.

jreback committed:
DOC: highlite to contributing.rst

evanpw committed:
BUG: null group spills into final group when grouping on a categorical

dr-leo committed:
Docs: add reference to pandaSDMX to the ecosystem

mortada committed:
DOC: improve fillna() doc for limit keyword (fixes #10002)

mortada committed:
kwarg for Categorical.fillna should be value instead of fill_value

mortada committed:
DOC: added fredapi module to ecosystem list

sinhrks committed:
ENH: Added str.normalize to use unicodedata.normalize

mortada committed:
DOC: use _shared_docs for fillna methods

cpcloud committed:
Network-ize a test

cpcloud committed:
Use explicit len dispatch to avoid overhead

cpcloud committed:
Improve perf

Commit:
Fix missing index name in test

cpcloud committed:
Use a fused type

jblackburne committed:
Changed a condition in tokenize_delimited to account for data chunks that start with newline.

Changed a condition in tokenize_delim_customterm to account for data chunks that start with terminator.

Added a unit test that fails in master and passes in this branch.

Moved new unit test in order to test all parser engines. Added GH issue number.

Added release note.

cpcloud committed:
Ensure object on stata

cpcloud committed:
Test that we do not accept unicode

cpcloud committed:
Use proper types so that we work with python3

cpcloud committed:
Better name for fused type

shoyer committed:
Fix TimedeltaIndex constructor fastpath when name is set

Fixes GH10025

shoyer committed:
DOC: mention other packages in IO docs (for now, just xray for netCDF)

As discussed in #5487.

This would also be a nice place to mention other packages that connect the
pandas DataFrames with other file formats. Right now, the only one I can think
of off-hand is `root_pandas` (#9378, CC @ibab), but I'm sure there are more.

Commit:
Updates to Google BigQuery connector (#9713, #8327)

commit 3d6fdc8751134f6e5bac700519358cf9a700aba1
Author: Chris Grinolds <cgrinolds@mac.com>
Date:   Wed Apr 29 21:43:10 2015 -0700

    Added missing imports that caused tests to fail

commit 6c345d8dbc0c36bd4369dd220eceb495ab5ff2c6
Author: Chris Grinolds <cgrinolds@zulily.com>
Date:   Wed Apr 29 16:23:01 2015 -0700

    Updated test suite to handle changes to gbq.py

commit f43e65f924d07f88bbbe308b9ea396b1f2a720f9
Author: Chris Grinolds <cgrinolds@zulily.com>
Date:   Wed Apr 29 08:46:00 2015 -0700

    Updated BigQuery connector to no longer use deprecated ```oauth2client.tools.run() (#8327)

commit ea92e200ff341a3025b06868a565b1eea506c4c2
Author: Chris Grinolds <cgrinolds@zulily.com>
Date:   Wed Apr 29 08:43:43 2015 -0700

    Import BigQuery dependencies on a per-method basis (#9713)

cpcloud committed:
pd.lib.infer_dtype now returns bytes in python 3 when appropriate

jreback committed:
disallow numeric setlike operations on CategoricalIndex (GH10039)

jreback committed:
BUG: provide deprecation warnings when using setlike operations on Indexes and lists (for +/-) (GH10038)

jreback committed:
COMPAT: windows test dtype adjustments

scari committed:
BUG: Resample BM/BQ adds extra index point #9756

mortada committed:
ENH: support StringMethods index and rindex

hayd committed:
ENH Add normalize method to Series dt accessor.

TomAugspurger committed:
DOC: add docstring for PeriodIndex.asfreq

jnmclarty committed:
DOC: infer_freq, inferred_freq

gdementen committed:
CLN: cleanup hashtable.pyx

remove unused variables and commented debug code
do not store result of kh_put* when it is not used
use khiter_t instead of int
rename val & max_val to count & max_count for readability & consistency

Commit:
Return correct subclass when slicing DataFrame.

mortada committed:
ENH: support __radd__ operation on Index (GH10083)

jreback committed:
cleanup test for GH10044

scari committed:
ENH: Series.resample performance with datetime64[ns] #7754

mortada committed:
ENH: support str translate for StringMethods

mortada committed:
DOC/CLN: fixed typos in timeseries.rst

sinhrks committed:
BUG: Series.align resets name when fill_value is specified

mortada committed:
DOC/CLN: docs fixes for Series.shift and DataFrame.shift

mortada committed:
BUG: Panel.from_dict does not set dtype when specified

jreback committed:
DOC: add warning for datareader changes in 0.17.0 (GH10004)

jreback committed:
BUG: Bug in grouping with multiple pd.Grouper where one is non-time based (GH10063)

jreback committed:
doc edits for GH10047

jorisvandenbossche committed:
DOC: remove use of deprecated index set operation (follow-up GH10042)

jorisvandenbossche committed:
Change Index repr to adjust to string length

Conflicts:
	pandas/tseries/base.py

use new format_data

updates

Fix detection of good width

more fixes

Change [

Conflicts:
	pandas/core/index.py

more fixes

revsised according to comments

jreback committed:
DOC/CI: add blosc to optional deps list (msgpack compression)

jreback committed:
DOC: fix in v0.16.1

rockg committed:
DOC: Note on PyTables index issue and additional Contributing refinements.

ogrisel committed:
FIX make take_nd support readonly array

jreback committed:
BUG: use memory view for indexer in take_2d / tests (GH10043)

sinhrks committed:
DOC: Fix v0.16.1 release note

mortada committed:
BUG: median() not correctly handling non-float null values (fixes #10040)

jorisvandenbossche committed:
DOC/DEPR: port pandas.rpy to rpy2 guide (GH9602)

jorisvandenbossche committed:
DOC/DEPR: add link to docs in deprecation warning

FvD committed:
Corrected typo in Grammar of Graphics

jreback committed:
TST: dtype comparison issue on windows for test_resample_index_points

Commit:
BUG: Use temp files in all pytables tests to fix (GH9992).

sinhrks committed:
DEPR: Deprecate str.split return_type #9847

jreback committed:
DOC: additional whatsnew

jreback committed:
TST: using - rather than difference in tools/plotting.py

cpcloud committed:
Release note [ci skip]

jorisvandenbossche committed:
DOC: consistent imports (GH9886) part II

jorisvandenbossche committed:
Index repr: allow unequal number of elements on one line

Inspired by numpy's array2string

floydwch committed:
fix the inconsistency between code and description

jreback committed:
DOC: prepare for 0.16.1 release
DOC: edits in v0.16.1 whatsnew

jreback committed:
more fixups

mortada committed:
BUG: Series.fillna() raises if given a numerically convertible string

jreback committed:
DOC: add v0.17.0.txt whatsnew (not in index yet though)

jreback commented:
@CoolRanch29 can you update

sinhrks committed:
BUG: Timestamp properties may return np.int

jreback committed:
DOC: missed merge in v0.16.1 whatsnew

mortada committed:
CLN: clean up unused imports

jreback committed:
RLS: v0.16.1 final

jreback committed:
initial commit for v0.17.0

jorisvandenbossche committed:
DOC: update docs regarding to return_type/expand (due to GH10085)

jorisvandenbossche committed:
DOC: fix some doc build errors

- wrap docstring -> raw indication needed for newlines characters
- using anonymous hyperlinks (two underscores instead of one) for preventing warning on same names
- some missing imports and whitespace

jorisvandenbossche committed:
DOC: last clean-up of whatsnew file 0.16.1

jorisvandenbossche committed:
DOC: update section on CategoricalIndex in categorical docs

jreback committed:
DOC: add v0.17.0 whatsnew to index

rosnfeld committed:
BUG: categorical doesn't handle display.width of None in Python 3 (GH10087)

TomAugspurger committed:
BUG: fix Series.plot label setting

mortada committed:
DOC: improve binary operator docs (fixes #10093)

mortada committed:
CLN: clean up unused imports part II

jreback committed:
DOC: fixup whatsnew

sinhrks committed:
BUG: Index.union cannot handle array-likes

jreback committed:
TST: windows tests for allow for int/long #10050

sinhrks committed:
TST: use compat.long in test_tslib for py3

mortada committed:
BUG: mean overflows for integer dtypes (fixes #10155)

ElDeveloper committed:
DOC: Reorder arguments in shared fillna docstring

The docstring listed 'method' before 'value' which is not consistent
with the order of the arguments when calling the method.

jreback committed:
DOC: fix blosc display in install.rst

jreback committed:
BUG: consistent datetime display format with < ms #10170

mortada committed:
DOC/CLN: period_range kwarg descriptions missing and other minor doc fixes

jreback commented:
@CoolRanch29 want to update?

sinhrks committed:
DOC: Add Index.difference to API

jakevdp committed:
DOC: minor doc fix for Series.append; indices can overlap

robintw committed:
DOC: Updated to mention axis='index' and axis='columns

Updated docs throughout DataFrame methods to mention
that axis can be set to 'index' or 'column' instead of 0 or 1
which improves readability significantly.

ajamian committed:
BUG: closes issue #7212 - side effect on passed columns list

CoolRanch29 committed:
Rebased

CoolRanch29 commented:
Okay, I believe I just rebased.

On Wed, May 20, 2015 at 7:06 AM, jreback <notifications@github.com> wrote:

> @CoolRanch29 <https://github.com/CoolRanch29> want to update?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/9350#issuecomment-103844130>.
>

robintw committed:
DOC: Added nlargest/nsmallest to API docs

Fixes #10145

jreback commented on doc/source/whatsnew/v0.16.1.txt:
```diff
@@ -80,3 +80,5 @@ Bug Fixes
 - Bug in ``Series.quantile`` on empty Series of type ``Datetime`` or ``Timedelta`` (:issue:`9675`)
 - Bug in ``where`` causing incorrect results when upcasting was required (:issue:`9731`)
+
+- Bug in ``ParserBase.convert_to_nd_arrays`` when called by ``DataFrame.read_fwf`` (:issue:`9266``) 
```
needs to be moved to 0.17.0. Say something like ``fixed bug in csv parsing when using a converting that specified uint8``

jreback commented:
See here for rebasing help. Pls squash to a single commit as well. http://pandas.pydata.org/pandas-docs/stable/contributing.html

mortonfox committed:
Update bq link

Commit:
BUG: plot doesnt default to matplotlib axes.grid setting (#9792)

Cleanup

Commit:
Close mysql connection in TestXMySQL to prevent tests freezing

bwillers committed:
BUG:  Holiday(..) with both offset and observance raises NotImplementedError #10217

GH10217

sinhrks committed:
BUG: SparseSeries.abs() resets name

sinhrks committed:
BUG: Series arithmetic methods incorrectly hold name

jreback commented:
can you rebase/squash

jreback committed:
DOC: move whatsnew from 0.17.0 -> 0.16.2

jreback committed:
DOC: add in whatsnew/0.17.0.txt

CoolRanch29 committed:
Moved bug fix to v0.17.0.txt
